model/task/db.go

```diff
@@ -2045,6 +2046,8 @@ func recalculateTimeTaken() bson.M {
 // GetTasksByVersion gets all tasks for a specific version
 // Query results can be filtered by task name, variant name and status in addition to being paginated and limited
 func GetTasksByVersion(ctx context.Context, versionID string, opts GetTasksByVersionOptions) ([]Task, int, error) {
+	ctx = utility.ContextWithAttributes(ctx, []attribute.KeyValue{attribute.String("db.aggregationName", "GetTasksByVersion")})
```

Comment:
Could it make sense to keep the attribute name in a package constant? Or even in [this global list](https://github.com/evergreen-ci/evergreen/blob/4d5718c8b92d7dd5955f29a0bb74cc6c8a0b5e74/globals.go#L463-L476)?

Reply:
Good catch! Done!